proposals/lip-0053.md

```diff
@@ -92,7 +92,7 @@ These points guarantee that the CCMs are always forwarded to the correct chains
 The [execution of cross-chain messages][lip-0049#apply] as part of the processing of a cross-chain update is done in several stages, in a way analogous to the execution of a transaction as part of the [processing of a block][lip-0055#block-processing-stages]:
 
 1. **Cross-chain message verification**: Each module can define protocol logic that verifies a CCM, possibly by accessing the state store. If an error occurs, the channel with the CCU sending chain is terminated and the CCU execution stops.
-2. **Cross-chain command verification**: The cross-chain command corresponding to the CCM `module`-`crossChainCommand` combination is verified. If an error occurs, the channel with the CCM sending chain is terminated and the CCM execution stops.
+2. **Cross-chain command verification**: The parameters are decoded using `decode` function and validated against the schema using `validateObjectSchema`. Then, the cross-chain command corresponding to the CCM `module`-`crossChainCommand` combination is verified. If an error occurs either during parameter validation or cross-chain command verification, the channel with the CCM sending chain is terminated and the CCM execution stops.
```

Comment:
If we finally remove the checks as I suggested, we should revert these changes as well

Reply:
Note that `validateFormat` only validates the root schema so it is still necessary in the `apply` function to validate the CCM parameters.